concepticondata/conceptlists/Tolmie-1884-211.tsv

```diff
@@ -0,0 +1,212 @@
+Tolmie-1884-211-35	35	Hand	1277	HAND
+Tolmie-1884-211-36	36	Fingers	1303	FINGER
+Tolmie-1884-211-37	37	Thumb	1781	THUMB
+Tolmie-1884-211-38	38	Nails	1896	FINGERNAIL OR TOENAIL
+Tolmie-1884-211-39	39	Body	1480	BODY
+Tolmie-1884-211-40	40	Chest	1592	CHEST
+Tolmie-1884-211-41	41	Belly	1251	BELLY
+Tolmie-1884-211-42	42	Female breasts	1402	BREAST
+Tolmie-1884-211-43	43	Leg	1297	LEG
+Tolmie-1884-211-44	44	Foot	1301	FOOT
+Tolmie-1884-211-45	45	Toes	1389	TOE
+Tolmie-1884-211-46	46	Bone	1394	BONE
+Tolmie-1884-211-47	47	Heart	1223	HEART
+Tolmie-1884-211-48	48	Blood	946	BLOOD
+Tolmie-1884-211-49	49	Town, village		
+Tolmie-1884-211-50	50	Chief	2394	CHIEF
+Tolmie-1884-211-51	51	Warrior		
+Tolmie-1884-211-52	52	Friend	1325	FRIEND
+Tolmie-1884-211-53	53	House	1252	HOUSE
+Tolmie-1884-211-54	54	Skin lodge		
+Tolmie-1884-211-55	55	Kettle	854	KETTLE
+Tolmie-1884-211-56	56	Bow	994	BOW
+Tolmie-1884-211-57	57	Arrow	977	ARROW
+Tolmie-1884-211-58	58	Axe, hatchet	2083	AXE (CLASSIFIER)
+Tolmie-1884-211-59	59	Knife	1352	KNIFE
+Tolmie-1884-211-60	60	Canoe	1970	CANOE
+Tolmie-1884-211-61	61	Moccasins		
+Tolmie-1884-211-62	62	Pipe	1124	PIPE
+Tolmie-1884-211-63	63	Tobacco	974	TOBACCO
+Tolmie-1884-211-64	64	Sky	1732	SKY
+Tolmie-1884-211-65	65	Sun	1343	SUN
+Tolmie-1884-211-66	66	Moon	1313	MOON
+Tolmie-1884-211-67	67	Star	1430	STAR
+Tolmie-1884-211-68	68	Day	1225	DAY (NOT NIGHT)
+Tolmie-1884-211-69	69	Night	1233	NIGHT
+Tolmie-1884-211-70	70	Morning	1339	MORNING
+Tolmie-1884-211-71	71	Evening	1629	EVENING
+Tolmie-1884-211-72	72	Spring	1537	SPRINGTIME
+Tolmie-1884-211-73	73	Summer	1342	SUMMER
+Tolmie-1884-211-74	74	Autumn	1276	AUTUMN
+Tolmie-1884-211-75	75	Winter	1471	WINTER
+Tolmie-1884-211-76	76	Wind	960	WIND
+Tolmie-1884-211-77	77	Thunder	1150	THUNDER
+Tolmie-1884-211-78	78	Lightning	1654	LIGHTNING
+Tolmie-1884-211-79	79	Rain	658	RAIN (PRECIPITATION)
+Tolmie-1884-211-80	80	Snow	784	SNOW
+Tolmie-1884-211-81	81	Fire	221	FIRE
+Tolmie-1884-211-82	82	Water	948	WATER
+Tolmie-1884-211-83	83	Ice	617	ICE
+Tolmie-1884-211-84	84	Earth, land		
+Tolmie-1884-211-85	85	Sea	1474	SEA
+Tolmie-1884-211-86	86	River	666	RIVER
+Tolmie-1884-211-87	87	Lake	624	LAKE
+Tolmie-1884-211-88	88	Valley	921	VALLEY
+Tolmie-1884-211-89	89	Prairie		
+Tolmie-1884-211-90	90	Hill, mountain	2118	MOUNTAIN OR HILL
+Tolmie-1884-211-91	91	Island	622	ISLAND
+Tolmie-1884-211-92	92	Stone, rock	2125	STONE OR ROCK
+Tolmie-1884-211-93	93	Salt	1274	SALT
+Tolmie-1884-211-94	94	Iron	621	IRON
+Tolmie-1884-211-95	95	Forest	420	FOREST
+Tolmie-1884-211-96	96	Tree	906	TREE
+Tolmie-1884-211-97	97	Wood	1803	WOOD
+Tolmie-1884-211-98	98	Leaf	628	LEAF
+Tolmie-1884-211-99	99	Bark	1204	BARK
+Tolmie-1884-211-100	100	Grass	606	GRASS
+Tolmie-1884-211-101	101	Pine	1740	PINE
+Tolmie-1884-211-102	102	Maize	506	MAIZE
+Tolmie-1884-211-103	103	Squash	864	PUMPKIN
+Tolmie-1884-211-104	104	Flesh, meat	2615	FLESH OR MEAT
+Tolmie-1884-211-105	105	Dog	2009	DOG
+Tolmie-1884-211-106	106	Buffalo	934	BUFFALO
+Tolmie-1884-211-107	107	Bear (black)		
```

Comment:
you did not link the bear because it is a black bear? I'd say: link to BEAR for now.